### PR TITLE
Updates for latest rustc (2015-03-20)

### PR DESCRIPTION
**Pull request by SkylerLipthay**
Everything should be pretty straightforward here. I'm very oblivious to `ParserAnyMacro`'s particulars, but simply copying from the [latest macro_rules.rs](https://github.com/rust-lang/rust/blob/92dd995e1705a8c398a5266c8483e2519e494f47/src/libsyntax/ext/tt/macro_rules.rs) seemed to work.

### CODE REVIEW

**Comment by autohuonw** [comment]
Thanks for the pull request, and welcome! You should hear from @huonw (or someone else) soon.

File: Cargo.toml [hunk 1]
```diff
@@ -23,6 +23,9 @@ default = []
 # examples/failure.rs test
 compile_error = []
 
+[dependencies]
+tempdir = "*"
```
Comment (huonw) [inline Cargo.toml]: Could you make this `tempdir = "0.3"`?

**Comment by huonw** [comment]
Thanks!

It's weird that travis is failing, I can't see why it would given this patch.

**Comment by SkylerLipthay** [comment]
Changed `Cargo.toml` with `tempdir = "0.3"` and forced a push.

I don't know if this helps with Travis, but locally this failed:

```
cargo build --verbose && cargo test --verbose && rustdoc -L ./target/deps --test README.md
```

...and this succeeded:

```
cargo build --verbose && cargo test --verbose && rustdoc -L ./target/debug/deps --test README.md
```

(Note `/debug`)

**Comment by huonw** [comment]
Oh, of course: cargo changed its directory structure. Thanks for tracking that down. Maybe you could [adjust the .travis.yml](https://github.com/huonw/external_mixin/blob/master/.travis.yml#L5) to use the `/debug/` version?

**Comment by SkylerLipthay** [comment]
Done!

**Comment by huonw** [comment]
Great, thanks.

Let's try this,

@autohuonw r+

**Comment by autohuonw** [comment]
:pushpin: Commit cfaa855 has been approved by `huonw`

**Comment by autohuonw** [comment]
:hourglass: Testing commit cfaa855 with merge e8c4d25...

**Comment by autohuonw** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/huonw/external_mixin/builds/55355129)

**Pull request merged**